### PR TITLE
Updated black holes

### PR DESCRIPTION
**Pull request by sarthaks-23**

### CODE REVIEW

File: theory/special stars/black holes.md [hunk 1]
```diff
@@ -7,3 +7,52 @@ nav_order: 3
...
+
+2. Time Dilation: As predicted by Einstein’s theory of general relativity, time slows down near a black hole’s event horizon. An observer watching an object fall into a black hole would see it slow down and freeze as it approaches the event horizon.
+
+3. [Gravitational Waves](../cosmology/gravitational%20waves.html): When two black holes merge, they produce ripples in space-time known as [gravitational waves](../cosmology/gravitational%20waves.html). These waves have been detected by observatories like LIGO, confirming Einstein’s predictions and providing new insights into black hole properties.
+
+### Black Hole Information Paradox
+
+One of the biggest unresolved puzzles in physics is the [black hole](#black-holes) information paradox. According to quantum mechanics, information cannot be destroyed, yet anything that falls into a black hole seems to be lost forever. This conflict with general relativity has sparked significant debate. Some theories suggest information may be encoded at the event horizon or could escape through Hawking radiation, a theoretical process by which black holes gradually lose mass.
```
Comment (Jadit19) [inline theory/special stars/black holes.md:56]: Please hyperlink the article on [black hole information paradox](https://astroclubiitk.in/2020-09-02-blackhole-paradox/) as a callout here

Comment (Jadit19) [inline theory/special stars/black holes.md:56]: Please hyperlink the article on [black hole information paradox](https://astroclubiitk.in/2020-09-02-blackhole-paradox/) as a callout here

File: theory/special stars/black holes.md [hunk 2]
```diff
@@ -7,3 +7,52 @@ nav_order: 3
 ---
 
 ## Black Holes
+
+A black hole is an astronomical object with a gravitational pull so strong that nothing, not even light, can escape it.
+
+### Structure of Black Holes
+![blackhole](../../assets/images/theory/special%20stars/black%20holes/blackhole.jpg)
```
Comment (Jadit19) [inline theory/special stars/black holes.md:14]: Please use reference from other pages on how to correctly insert credits

File: theory/special stars/black holes.md [hunk 3]
```diff
@@ -7,3 +7,52 @@ nav_order: 3
...
+
+4. Alongside this accretion process, [black holes](#black-holes) can also produce powerful jets—streams of high-energy particles ejected perpendicularly from the accretion disk, driven by the intense magnetic fields generated in these extreme environments.
+
+5. ontrary to common belief, [black holes](#black-holes) are not empty spaces but rather regions filled with highly compressed matter, creating immense gravitational forces that warp space and time around them.
+
+### Formation of Black Holes
+
+[Black holes](#black-holes) typically form from the remnants of massive stars that have exhausted their nuclear fuel. When a star depletes its energy, it undergoes a supernova explosion, ejecting its outer layers into space. If the remaining core is massive enough—at least three times the mass of the Sun—it will collapse under its own gravity, forming a black hole. Smaller stars, like the Sun, do not become black holes. Instead, they evolve into white dwarfs or [neutron stars](neutron%20stars.html).
```
Comment (Jadit19) [inline theory/special stars/black holes.md:30]: Can you reference the page on [stellar evolution](https://wiki.astroclubiitk.in/theory/stellar%20physics/evolution.html) here?

**Review by Jadit19 — CHANGES_REQUESTED** [review]

**Comment by sarthaks-23** [comment]
I have updated the needed changes. Please verify from your side.

File: theory/special stars/black holes.md (hunk 2 above)
Comment (sarthaks-23) [reply theory/special stars/black holes.md:14]: Have done the changes.

File: theory/special stars/black holes.md (hunk 3 above)
Comment (sarthaks-23) [reply theory/special stars/black holes.md]: Have done the changes.

**Pull request merged**